streams/src/main/java/org/apache/kafka/streams/kstream/KStreamBuilder.java

```diff
@@ -51,17 +50,17 @@
  * @deprecated Use {@link org.apache.kafka.streams.StreamsBuilder StreamsBuilder} instead
  */
 @Deprecated
-public class KStreamBuilder extends TopologyBuilder {
+public class KStreamBuilder extends org.apache.kafka.streams.processor.TopologyBuilder {
```

Comment:
To avoid importing the `org.apache.kafka.streams.processor.TopologyBuilder` which will introduce a WARNING during compilation; instead we will explicitly refer to the class within the class whenever we have to.